gax-java/pom.xml

```diff
@@ -96,13 +96,13 @@
         <version>${auto-value.version}</version>
       </dependency>
       <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.13.2</version>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
+        <version>4.11.0</version>
```

Comment:
Why not using mockito bom?

Reply:
I was not aware of a mockito bom, sounds like a good idea. Also, I think adding the bom to this pom should fine, because [gax-parent](https://github.com/googleapis/sdk-platform-java/blob/main/gax-java/pom.xml) is not included in the gapic-generator-java-bom, only [gax-bom](https://github.com/googleapis/sdk-platform-java/blob/main/gax-java/gax-bom/pom.xml) is. Is that correct?